rest-api-spec/test/cluster.put_settings/10_basic.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 setup:
   - skip:
-      version: 0 - 999
+      version:  " - "
```

Comment:
Another question is if we want to have a "pending" test like this in the suite at all? Again, if yes, I'd like to have something semantic, like a `pending: true` flag, not a `skip` which needs to be deciphered?